src/styles/card.css

```diff
@@ -0,0 +1,81 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+  margin: 0 auto 40px;
+}
+
+.card__item-name {
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  font-size: 10px;
+  font-weight: 400;
+  color: #616070;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  align-items: center;
+  line-height: 18px;
+}
+
+.card__star-reviews {
+  font-size: 10px;
+  margin: 0;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__price-name {
+  font-size: 12px;
+  color: #616070;
+}
+
+.card__price-number {
+  font-size: 16px;
+  font-weight: 600;
+  color: #060b35;
+}
+
+.card__button {
+  height: 40px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  color: white;
+  border: 1px solid #00acdc;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 14px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  line-height: 16px;
+  transition: 0.3s;
```

Comment:
You need to specify properties here, that can be changeable for this element (for now it uses all by default)